src/components/SchedulePage/Timeslot.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+import Divider from '@material-ui/core/Divider';
+import Event from '../SchedulePage/Event';
+import ScheduleRoundedIcon from '@material-ui/icons/ScheduleRounded';
+
+const useStyles = makeStyles(theme => ({
+	time: {
+		fontFamily: theme.typography.fontFamily,
+		marginLeft: 4
+	},
+	timeContainer: {
+		display: 'flex',
+		alignItems: 'center'
+	}
+}));
+
+const formatTime = time => {
+	let hours = time.getHours();
+	let minutes = time.getMinutes();
+	const ampm = hours >= 12 ? 'PM' : 'AM';
+	hours %= 12;
+	hours = hours ? hours : 12; // the hour '0' should be '12'
+	minutes = minutes < 10 ? '0' + minutes : minutes;
+	const strTime = hours + ':' + minutes + ' ' + ampm;
+	return strTime;
+};
```

Comment:
1. Change `time` to `date`, as the input is a `Date` object.

2. Even better, use [`Intl.DateTimeFormat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat).

   ```js
   const timeFormatter = new Intl.DateTimeFormat('en-US', { timeStyle: 'short' });
   // ...
   <Typography className={classes.time}>{timeFormatter.format(time)}</Typography>
   ```